src/CoreAnimation/CACompat.cs

```diff
@@ -97,6 +97,22 @@ public CAAnimationStateEventArgs (bool finished)
 		}
 		public bool Finished { get; set; }
 	}
+
+	[Obsolete ("Type removed in Xcode 9 (rdar #33590997).")]
```

Comment:
Minor but ideally spell it fully: radar (:
Now I don't think there's much value in mentioning the radar number in the attribute because this isn't public. If we had it on open radar maybe? https://openradar.appspot.com/search?query=33590997